omc3/model/accelerators/lhc.py

```diff
@@ -132,7 +132,7 @@ def get_parameters():
             name="year",
             type=str,
             required=True,
-            choices=("2012", "2015", "2016", "2017", "2018","2021","2022", "hllhc1.3"),
+            choices=("2012", "2015", "2016", "2017", "2018", "2022", "hllhc1.3"),
```

Comment:
why was 2021 added just two days before to be removed now again?

Reply:
Because I added it since I thought it was the way the optics repository was setup but in the end it wasn't so removing it is the right thing.